### DISCUSSION TITLE
Add a "view all" button to the book strip view in Recommended page and search results

### DISCUSSION THREAD

**Discussion by zpaolo11x** [gotson/komga#1514] [Ideas]
### Describe your suggested feature

Recommended page shows strips of covers for Recently Added, Recently Read etc, search results show similar strips for Series and Books. Besides the arrows to scroll the strips left and right, it would be practical to have a "view all" button so that one can see all the items in one of those strips in the usual book cover UI.

### Acknowledgements

- [X] I have searched the existing issues and this is a new ticket, **NOT** a duplicate or related to another open issue.
- [X] I have written a short but informative title.
- [X] I have updated the app to the latest version.
- [X] I will fill out all of the requested information in this form.

**Comment by gotson** [comment]
🎉 This issue has been resolved in `1.26.0` ([Release Notes](https://github.com/gotson/komga/releases/tag/1.26.0))
